dev/provider_packages/prepare_provider_packages.py

```diff
@@ -1598,7 +1598,9 @@ def copy_readme_and_changelog(provider_package_id: str, backport_packages: bool)
     package_format = os.environ.get("PACKAGE_FORMAT", "wheel")
     print(f"Building backport package: {_provider_package} in format ${package_format}")
     copy_readme_and_changelog(_provider_package, BACKPORT_PACKAGES)
-    command = ["python3", "setup.py", "--version-suffix-for-pypi", suffix]
+    command = ["python3", "setup.py"]
+    if suffix != "":
+        command.extend(['egg_info', '--tag-build', suffix])
     if package_format in ['sdist', 'both']:
         command.append("sdist")
     if package_format in ['wheel', 'both']:
```

Comment:
This will add `bdist_wheel` by default.

```
python3 setup.py egg_info --tag-build rc1 bdist_wheel
```

for example

Reply:
Yeah. That was intentional. I want `bdist_wheel` to be the default format which can be overridden by the `--package-format` flag. Anything wrong here?

Reply:
Based on Ash's comment on https://github.com/apache/airflow/pull/13020#discussion_r541099194, we either need one of `egg_info` (both), `wheel` or `bdist_wheel` right?

With the above code we will have both `egg_info` and `bdist_wheel` :

```
python3 setup.py egg_info --tag-build rc1 bdist_wheel
```

Reply:
No no. We need `egg_info --tag-build x`  and then either one or both of (bdist_wheel, sdist). 
The egg_info is really pre-requisite for both..

BUT I did found another issue with SVN rename while checking (it was only working when `--package-format both` was specified). Fixed it.